### PR TITLE
Rebase to alpine 3.17 with php8.1 (master)

### PR DESCRIPTION
**Pull request by nemchik**

### CODE REVIEW

**Comment by thespad** [comment]
CI is failing, looks like at least `/app/www/bootstrap/cache` needs to be writable, possibly other locations.

**Comment by nemchik** [comment]
Went through a few adjustments. CI is now passing. Build works (tested a new install).

**Review by thespad — APPROVED** [review]

File: root/etc/s6-overlay/s6-rc.d/init-heimdall-config/run [hunk 1]
```diff
@@ -35,22 +24,27 @@ for i in "${symlinks[@]}"; do
...
+    if [[ -d /config/www/"$(basename "${i}")" ]]; then
+        lsiown abc:abc /config/www/"$(basename "${i}")"
+    fi
 done
 
+# set permissions on new container
+if [[ ! -f /config/www/searchproviders.yaml ]]; then
+    echo "New container detected, setting permissions"
+    lsiown -R abc:abc \
+        /app \
```
Comment (thespad) [inline root/etc/s6-overlay/s6-rc.d/init-heimdall-config/run:36]: The performance on this is going to be terrible because of how many files are involved. Shouldn't need to chown the whole /app.

Comment (nemchik) [reply root/etc/s6-overlay/s6-rc.d/init-heimdall-config/run:36]: This behavior is what was done before this PR, but the conditions to trigger it were different because the app was extracted during init and then the `chown` was run. This PR includes the app in the image and only runs `lsiown` if it is a new install (when `/config/www/searchproviders.yaml` does not exist. it is created later in the init to prevent future `lsiown` runs).

I tried only running `lsiown` on `/app/www/bootstrap/cache` which then resulted in a php error indicating permissions were incorrect, but no indication of what file/folder was causing the issue.

tldr; this PR does not introduce new slowness (performance issues) that does not already exist in master. There may be a slight performance increase by having the app baked into the image (not having to extract it during init).

Comment (nemchik) [reply root/etc/s6-overlay/s6-rc.d/init-heimdall-config/run]: https://github.com/linuxserver/docker-heimdall/pull/120/commits/96d2cfabc4f74958e03987f21e4953e1a450cb56 should fix it. Not really an improvement over the extract method, but we use the `-tmp` method in other containers, so hopefully this will be consistent.
